package.json

```
@@ -2,7 +2,12 @@
   "name": "@react-native-community/geolocation",
   "version": "1.3.0",
   "description": "React Native Geolocation Module for iOS and Android",
-  "main": "js/index.js",
+  "react-native": "js/index.js",
+  "main": "web/commonjs/index.js",
+  "files": [
+    "web/",
+    "js/"
```

From experience with the NetInfo module, you also need to include the `android`, `ios`, and `.podspsec` files or everything will break.

This should fix it:

```suggestion
    "js/",
    "/android",
    "!/android/build",
    "/ios",
    "/*.podspec"
```